src/views/SystemIntake/index.tsx

```diff
@@ -66,11 +64,19 @@ export const SystemIntake = ({ match }: SystemIntakeProps) => {
   };
 
   const dispatchSave = () => {
-    if (id && formikRef.current.dirty) {
-      dispatch(saveSystemIntake(id, formikRef.current.values));
+    if (formikRef.current.dirty) {
+      dispatch(saveSystemIntake(formikRef.current.values));
     }
   };
 
+  useEffect(() => {
+    const getSystemIntakesEffect = async (): Promise<void> => {
+      dispatch(getSystemIntakes());
+      await setExistingIntakesLoaded(true);
```

Comment:
I'm not totally sure this `await` statement does anything. This should function exactly the same because the `await` isn't waiting for an action to be complete.

I think you're trying to set `setExistingIntakesLoaded` to `true` after the API fetches a user's intakes. You'd need to set some sort of loading flag in redux and set it to `true` after the API request is complete.